src/components/Week.tsx

```diff
@@ -1,17 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
 import DailyChecklist from './DailyChecklist';
 import './Week.global.css';
+import noteService from '../services/notes';
 
 const Week = () => {
   const d = new Date();
   const lastSunday = new Date(d.setDate(d.getDate() - d.getDay()));
   const weekTitle = `Week of ${new Intl.DateTimeFormat('en-US', {
     month: 'long',
   }).format(lastSunday)} ${lastSunday.getDate()}, ${lastSunday.getFullYear()}`;
+  const [fakeReload, setReload] = useState<boolean>(false);
```

Comment:
Quick question - why is this 'fakeReload'? Also if we wanna be stingy with naming schemes, 'setFakeReload'? :joy:

Reply:
Because it's not really reloading the page, just a work around to only reload a few components! 😂

Reply:
Check your passive-aggressiveness at the door.